roles/rsyslog/roles/output_roles/files_forwards/defaults/main.yaml

```diff
@@ -0,0 +1,193 @@
+---
+# Files and forwards output configuration
+# ---------------------------------
+
+# Files and Forwards Rsyslog output configuration rules
+# ---------------------------------
+rsyslog_files_and_forwards_output_rules:
+  - '{{ rsyslog_conf_files_output_modules }}'
+  - '{{ rsyslog_conf_files_output_rules }}'
+  - '{{ rsyslog_conf_forwards_output_rules }}'
+  - '{{ rsyslog_conf_remote_templates }}'
+
+# rsyslog_conf_files_output_modules:
+rsyslog_conf_files_output_modules:
+
+  - name: 'output-modules'
+    type: 'modules'
+    sections:
+
+      - comment: 'Log messages into files with timestamp format either traditional or default'
+        options: |-
+          {% if rsyslog_basics_use_traditional_timestamp_format|default(true) %}
+          module(load="builtin:omfile" Template="RSYSLOG_TraditionalFileFormat")
+          {% else %}
+          module(load="builtin:omfile")
+          {% endif %}
+
+# rsyslog_conf_files_output_rules:
+#
+# A set of default ``rsyslog`` options which store local system logs in files
+# located in `{{rsyslog_system_log_dir}}/` directory. This is mostly the
+# same as the default ``rsyslogd`` configuration provided in the installations.
+rsyslog_conf_files_output_rules:
+
+  - name: 'output-files'
```

Comment:
@nhosoi What is the benefit of having this in the rsyslog.d/ instead of the rsyslog.conf.
If we want to keep this here we need a way to filter messages that should not go through these filters.

Reply:
@sradco:
> @nhosoi What is the benefit of having this in the rsyslog.d/ instead of the rsyslog.conf.
> If we want to keep this here we need a way to filter messages that should not go through these filters.

Our final goal is to manage the rsyslog configuration from LSR/Logging.  That includes logging the system logs into the local files, forwarding them to the rsyslog server on the remote host or other servers or message queue, etc.  I thought all the parts are preferable to handle in the same way instead of making the output to the local file specially.  

Also, we'd like to make the upgrade the configuration as easy as possible.  For instance, if a customer modifies rsyslog.conf as they wish, it might be hard for us to keep track of the changes and apply them to the new version of rsyslog.conf.  The update issue has not been fully solved yet, though.  I'm afraid all in one rsyslog.conf could make the task harder.

Reply:
@nhosoi I agree with you, but we need a way to only collect to the files the relevant logs. 
Currently viaq/ovirt and any other logs will also be saved to the files (usually /var/log/messages).
This is not the desired state, unless for debugging purposes for example.

Reply:
> @nhosoi I agree with you, but we need a way to only collect to the files the relevant logs.
> Currently viaq/ovirt and any other logs will also be saved to the files (usually /var/log/messages).
> This is not the desired state, unless for debugging purposes for example.

+1
This task is one of such requirements.
https://projects.engineering.redhat.com/browse/RHELPLAN-32351
In the ovirt case, what could be the source of the distribution process?  Tag?  Or some string embedded in the message?  Could you please open a task with the specific requirement in https://projects.engineering.redhat.com/browse/RHELPLAN-27211?